cypress/integration/item/share/categories.spec.js

```diff
@@ -0,0 +1,98 @@
+import {
+  buildShareButtonId,
+  CATEGORIES_SELECTION_VALUE_SELECTOR,
+  SHARE_ITEM_CATEGORY_DISCIPLINE,
+  SHARE_ITEM_CATEGORY_LEVEL,
+} from '../../../../src/config/selectors';
+import { buildItemPath } from '../../../../src/config/paths';
+import {
+  ITEM_WITH_CATEGORIES,
+  SAMPLE_CATEGORIES,
+} from '../../../fixtures/categories';
+import { DEFAULT_TAGS } from '../../../fixtures/itemTags';
+import { ITEM_LOGIN_ITEMS, PUBLISHED_ITEM } from '../../../fixtures/items';
+import { MEMBERS, SIGNED_OUT_MEMBER } from '../../../fixtures/members';
+
+const openShareItemTab = (id) => {
+  cy.get(`#${buildShareButtonId(id)}`).click();
+};
+
+const findCategoryNameById = (id) =>
+  SAMPLE_CATEGORIES.find((entry) => entry.id === id)?.name;
+
+export const deleteOption = () => {
+  cy.get(`#${SHARE_ITEM_CATEGORY_LEVEL}`).click();
+  cy.get('.MuiAutocomplete-popper li[data-option-index="0"]').click();
```

Comment:
You are always deleting the first one? In that case either precise it in the function name, or use a variable to define which option to delete. 
Also, it will fail if there's 2 autocomplete components in the same page, could you use a "context" to select the component more precisely?